### PR TITLE
Fix "force private" logic

### PR DESCRIPTION
**Pull request by wxiaoguang**
When creating a repo, the "FORCE_PRIVATE" config option should be respected, `readonly` doesn't work for checkbox, so it should use `disabled` attribute.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
There is one more incorrect readonly checkbox, last line here:

```
$ rg checkbox.+readonly
templates/repo/migrate/gogs.tmpl:106:               <input name="private" type="checkbox" checked readonly>
templates/repo/migrate/gitlab.tmpl:103:               <input name="private" type="checkbox" checked readonly>
templates/repo/migrate/onedev.tmpl:92:                <input name="private" type="checkbox" checked readonly>
templates/repo/migrate/github.tmpl:106:               <input name="private" type="checkbox" checked readonly>
templates/repo/migrate/git.tmpl:66:               <input name="private" type="checkbox" checked readonly>
templates/repo/migrate/gitea.tmpl:104:                <input name="private" type="checkbox" checked readonly>
templates/repo/migrate/gitbucket.tmpl:108:                <input name="private" type="checkbox" checked readonly>
templates/repo/migrate/codebase.tmpl:92:                <input name="private" type="checkbox" checked readonly>
templates/repo/create.tmpl:53:                <input name="private" type="checkbox" checked readonly>
templates/repo/settings/options.tmpl:33:              <input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}{{if and $.ForcePrivate .Repository.IsPrivate}} readonly{{end}}>

**Comment by wxiaoguang** [comment]
`repo/settings/options.tmpl` is special, done in new commit

File: templates/repo/settings/options.tmpl [hunk 1]
```diff
@@ -28,9 +28,10 @@
 						<label>{{ctx.Locale.Tr "repo.visibility"}}</label>
 						<div class="ui checkbox" {{if and (not .Repository.IsPrivate) (gt .Repository.NumStars 0)}}data-tooltip-content="{{ctx.Locale.Tr "repo.stars_remove_warning"}}"{{end}}>
 							{{if .IsAdmin}}
-							<input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}>
+								<input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}>
 							{{else}}
-							<input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}{{if and $.ForcePrivate .Repository.IsPrivate}} readonly{{end}}>
+								<input name="private" type="checkbox" {{if .Repository.IsPrivate}}checked{{end}}{{if and $.ForcePrivate .Repository.IsPrivate}} disabled{{end}}>
+								{{if and .Repository.IsPrivate $.ForcePrivate}}<input type="hidden" name="private" value="{{.Repository.IsPrivate}}">{{end}}
```
Comment (silverwind) [inline templates/repo/settings/options.tmpl:34]: Is it fine to have two form fields with `name=private`?

Comment (wxiaoguang) [reply templates/repo/settings/options.tmpl:34]: `disabled` means it won't be submitted, that's why it needs an extra one here.

**Review by delvh — APPROVED** [review]

**Pull request merged**